auth_supplier/views/auth_supplier_view.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
```

Comment:
Change this to <odoo> to use the new "cool" tag (and to avoid the need to change it when the other tag will be deprecated)